ion/bitstream.go

```diff
@@ -660,6 +660,11 @@ func (b *bitstream) ReadBytes() ([]byte, error) {
 		panic("not a lob")
 	}
 
+	// A0 and 90 are special cases, denoting an empty blob and an empty clob respectively, with b.len == 0.
+	if b.len == 0 {
+		return make([]byte, 0), nil
```

Comment:
~~Rather than instantiating an empty byte array each time, could we return a constant?~~

Go doesn't allow `const` slices. :)

It looks like [we can avoid a small amount of overhead](https://stackoverflow.com/q/29164375) by using an empty slice literal instead of calling `make`, however.

```suggestion
		return []byte{}, nil
```